j2k/testData/fileOrElement/kotlinApiAccess/CorrectNullabilityDetected.kt

```diff
@@ -3,13 +3,13 @@ import kotlinApi.*
 //TODO: Formatter works incorrectly
 internal class A {
     fun foo(c: KotlinClass): Int {
-        return c.nullableProperty!!.length
-        +c.property.length
-        +KotlinClass.nullableStaticVar!!
-        +KotlinClass.staticVar
-        +KotlinClass.nullableStaticFun(1)!!
-        +KotlinClass.staticFun(1)
-        +nullableGlobalFunction("")!!.length
-        +globalFunction("").length
+        return (c.nullableProperty!!.length
```

Comment:
After changes TODO comment above your code should be removed. You've already fixed it.